bokeh/tests/test_io.py

```diff
@@ -152,7 +152,36 @@ def test_missing_title(self, mock_warn):
         self.assertEqual(mock_warn.call_args[1], {})
 
 class Test_SaveHelper(DefaultStateTester):
-    pass
+
+    @patch('io.open')
+    @patch('bokeh.io.standalone_html_page_for_models')
+    def test_obj_arg_is_document(self, mock_standalone_html_page, mock_io_open):
+        obj = Document()
+        filename, resources, title = io._get_save_args(io._state, "filename", "resources", "title")
+        io._save_helper(obj, filename, resources, title, True)
+
+        self._check_func_called(mock_standalone_html_page,
+                                (obj, resources, title),
+                                {})
+        self._check_func_called(mock_io_open,
```

Comment:
This part of the test is "not great". It'd be better to have `io.open` generate a tempfile then check the content after writing to it, but it couldn't get it to work.